lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -380,7 +384,7 @@ def inner(self, default_store, module_count, mongo_calls, sql_queries, *args, **
 
     @ddt.data(
         (ModuleStoreEnum.Type.mongo, 3, 4, 31),
-        (ModuleStoreEnum.Type.split, 3, 13, 31),
+        (ModuleStoreEnum.Type.split, 3, 12, 31),
```

Comment:
I remember you guys talking about this, just wondering what the cause was that reduced the queries here?